sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/rest_client.hpp

```diff
@@ -921,6 +919,17 @@ namespace Azure { namespace Storage { namespace Blobs {
          */
         std::string LeaseId;
       };
+      struct BlobName final
+      {
+        /**
+         * Indicates if the blob name is encoded.
```

Comment:
"encoded" implies that there is an encoding. What is the encoding for the blob name? UTF-8? UTF-16? Base64? Base64url, DER? Something else?

Reply:
Ok, reading further in the code, it appears that this is URL encoding. As such, IMHO the name should be "UrlEncoded" or something similar.

Reply:
It's in `_detail` namespace so it doesn't really matter. The name is defined in swagger file, which is not maintained by our team. Making a change to swagger will break all other languages.

Reply:
That's fair.  I didn't realize that this was generated code.